mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -56,31 +72,17 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
 
     # get a list of timeSeriesStats output files from the streams file,
     # reading only those that are between the start and end dates
-    startDate = config.get('time', 'timeseries_start_date')
+    #startDate = config.get('time', 'timeseries_start_date')
+    startDate = '0001-01-01_00:00:00'
     endDate = config.get('time', 'timeseries_end_date')
     streamName = streams.find_stream(streamMap['timeSeriesStats'])
     infiles = streams.readpath(streamName, startDate=startDate,
                                endDate=endDate)
     print 'Reading files {} through {}'.format(infiles[0], infiles[-1])
-
-    casename = config.get('case', 'casename')
-    ref_casename_v0 = config.get('case', 'ref_casename_v0')
-    indir_v0data = config.get('paths', 'ref_archive_v0_ocndir')
-
-    compare_with_obs = config.getboolean('ohc_timeseries', 'compare_with_obs')
-
-    plots_dir = config.get('paths', 'plots_dir')
-
-    yr_offset = config.getint('time', 'yr_offset')
-
-    N_movavg = config.getint('ohc_timeseries', 'N_movavg')
-
-    regions = config.getExpression('regions', 'regions')
-    plot_titles = config.getExpression('regions', 'plot_titles')
-    iregions = config.getExpression('ohc_timeseries', 'regionIndicesToPlot')
+    print '  Warning: OHC time series always start from simulation year 1'
```

Comment:
@xylar: here is the warning you were suggesting. But it would obviously be not necessary if we chose to just recalculate mean_yr1.